libbeat/management/status/status.go

```diff
@@ -40,6 +40,8 @@ const (
 	Stopping
 	// Stopped is status describing unit is stopped.
 	Stopped
+	//Healthy is status describing unit is healthy.
```

Comment:
We need not introduce this new const. The RUNNING status is for telling that the serivce is healthy. Lets use that only